templates/channels/channel_configuration.haml

```diff
@@ -627,6 +627,38 @@
       https://{{ domain }}{% url 'handlers.viber_handler' 'receive' object.uuid %}
 
 
+  -elif object.channel_type == 'LN'
+    %h4
+      -blocktrans
+        To finish the configuration of Line channel you'll need to set the following callback URL in the Line Bot settings page, following the steps below:
+
+    .info
+
+      %p
+
+        %ol.line-steps
+          %li
+            -blocktrans
+              Configure "Callback URL" in the channel page (the same page which get the information Channel Secret and Channel Access Token) by clicking on the "Edit" button, filling the field "webhook URL" and pressing on the "Save" button.
+
+          %li
+            -blocktrans
+              Enter also the IP addresses in the "Server IP Whitelist" menu with the list of addresses that will be displayed after the correct filling out the form below.
```

Comment:
`Fill the IP addresses in the "Server IP Whitelist" with the list of addresses displayed below.`